hazelcast-jet-sql/src/test/java/com/hazelcast/jet/sql/SqlTestSupport.java

```diff
@@ -120,6 +120,17 @@ public static void assertRowsAnyOrder(String sql, Collection<Row> expectedRows)
         assertThat(actualRows).containsExactlyInAnyOrderElementsOf(expectedRows);
     }
 
+    public static void assertContainsOnlyOneOfRows(String sql, Row... expectedRows) {
+        assertContainsOnlyNOfRows(sql, 1, expectedRows);
+    }
+
+    public static void assertContainsOnlyNOfRows(String sql, int n, Row... expectedRows) {
```

Comment:
```suggestion
    /**
     * Asserts that the result of {@code sql} contains a subset of {@code expectedRows}, but
     * only a subset of them with size of {@code subsetSize}.
     */
    public static void assertContainsSubsetOfRows(String sql, int subsetSize, Row... expectedRows) {
```
I'd also consider moving this method to `SqlLimitTest`, not sure how useful it's elsewhere.